docs/personal/DALE_CHANGES.md

```
@@ -0,0 +1,208 @@
+# Dale's XcodeBuildMCP Fork - Change Log
+
+**Personal Fork:** https://github.com/carmandale/XcodeBuildMCP  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)